src/main/scala/shine/OpenCL/ParallelismLevel.scala

```diff
@@ -5,3 +5,6 @@ case object WorkGroup extends ParallelismLevel
 case object Global extends ParallelismLevel
 case object Local extends ParallelismLevel
 case object Sequential extends ParallelismLevel
+
+case object Warp extends ParallelismLevel
+case object Lane extends ParallelismLevel
```

Comment:
here as well it would be good to move this parallelism level to a common package used by both OpenCL and CUDA.

Reply:
mentioned in #106